Components/toast.css

```diff
@@ -0,0 +1,50 @@
+.bat-toast {
+    padding: 0.5rem 1rem;
+    background-color: #333;
+    /* there's no need for this color anywhere, that's why just using hex */
+    margin: auto;
+    color: var(--white-only);
+    display: flex;
+    justify-content: space-between;
+    font-size: 14px;
+    max-width: 400px;
```

Comment:
use consistent units

Reply:
I'll keep it in mind from next time. thanks.